compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/Passes.cpp

```diff
@@ -581,6 +584,17 @@ void addMLIRAIRAIELoweringPasses(OpPassManager &passManager, bool packPeel) {
 
   passManager.addPass(xilinx::airrt::createAIRRtToNpuPass());
   passManager.addPass(createCanonicalizerPass());
+
+  // Now lower using the AIE passes used by the mlir-air/mlir-aie flow.
+  passManager.addPass(createLowerAffinePass());
+  OpPassManager &devicePM = passManager.nest<xilinx::AIE::DeviceOp>();
+  devicePM.addPass(createAIEAssignLockIDsPass());
+  devicePM.addPass(createAIEObjectFifoStatefulTransformPass());
+  devicePM.addPass(createAIEAssignBufferDescriptorIDsPass());
+  devicePM.addPass(createAIEAssignBufferAddressesBasicPass());
+  devicePM.addPass(createAIEPathfinderPass());
+  devicePM.addPass(createAIELocalizeLocksPass());
+  passManager.addPass(createConvertSCFToCFPass());
```

Comment:
In order to keep the two paths separate (until we're ready to shift to using only `amd-aie-direct`) you can't actually use the in-tree passes. To put it simply: if you put this here today, tomorrow I will have to revert this commit because tomorrow (or very soon) I will be removing/deleting those passes.

Reply:
So it's very awkward I know but if you want to do this you basically need to restore the CMakes as done here https://github.com/nod-ai/iree-amd-aie/pull/421/files and then write `devicePM.addPass(AIE::createAIEPathfinderPass());` etc.

Reply:
wait, I think these are in tree passes no?

Reply:
ah you dont want me to use in-tree got it. then I have to make my own lib for it..

Reply:
Yes what you currently have written are calls to the "creators" of the in-tree passes and what I'm saying is tomorrow (soon) those will no longer exist and this code will break.